.github/publish-images/action.yaml

```diff
@@ -57,6 +56,14 @@ runs:
         IMG_BASE: ${{ inputs.image_registry }}
         ADDITIONAL_TAGS: ${{ inputs.additional_tags }}
 
+    - name: Build E2E
+      shell: bash
+      run: |
+        make e2e-test-image
+      env:
+        VERSION: ${{ inputs.version }}
+        IMG_BASE: ${{ inputs.image_registry }}
+
```

Comment:
@vprashar2929 why are we buidling the image if that isn't being used anywhere in CI or need to be pushed? 🤔 . Is to to ensure the image can be built?

Reply:
Yeah exactly.

Reply:
could you please add a note 🙏 ?